packages/react/src/components/table/Table.tsx

```diff
@@ -37,6 +37,11 @@ type Header = {
 };
 
 export interface TableCustomTheme {
+  /**
+   * Custom background color for the table.
```

Comment:
Maybe add: "... for the table rows or content"?

Reply:
- [x] fix comment